bin/idam-client-setup.sh

```diff
@@ -10,7 +10,7 @@ XUI_CLIENT_ID="xui_webapp"
 XUI_CLIENT_SECRET="xui_webapp_secret"
 BIN_FOLDER=$($(dirname "$0")/probate-dev-env-realpath)
 
-authToken=$(curl -v -H 'Content-Type: application/x-www-form-urlencoded' -XPOST "${IDAM_URI}/loginUser?username=idamOwner@hmcts.net&password=Ref0rmIsFun" | jq -r .api_auth_token)
+authToken=$(curl -s -X POST "${IDAM_URI}/loginUser" -H "accept: application/json" -H "Content-Type: application/x-www-form-urlencoded" -d "password=Ref0rmIsFun&username=idamOwner@hmcts.net" | docker run --rm --interactive stedolan/jq -r .api_auth_token)
```

Comment:
Wouldn't it be better to keep the verbose option?

Reply:
I got the silent (-s) version and the change in command from the swagger when i tried this endpoint . Hence used the same . having the -v was not of any great help either, It doesn't impact and this is fairly a straight forward curl i wouldn't bother much to remove verbose option .